src/components/app/data/services/bffs.test.ts

```diff
@@ -116,8 +120,13 @@ const mockBaseLearnerBFFResponse = {
   warnings: [],
 };
 
-const mockBFFDashboardResponse = {
+const mockBaseBFFResponse = {
   ...mockBaseLearnerBFFResponse,
+  ...mockSecuredAlgoliaMetadata,
```

Comment:
nit: I might recommend including `mockSecuredAlgoliaMetadata` as part of `mockBaseLearnerBFFResponse` since it part of the base BFF response.

For context, I believe the original intent for `mockBaseLearnerBFFResponse` was to effectively be the `mockBaseBFFResponse`.